synapse/models/dns.py

```diff
@@ -101,211 +107,76 @@ def getModelDefs(self):
                     ('email', ('inet:email', {}), {'ro': 1,
                          'doc': 'The email address (RNAME) returned in the SOA record.'}),
                 )),
+
                 ('inet:dns:txt', {}, (
                     ('fqdn', ('inet:fqdn', {}), {'ro': 1,
                          'doc': 'The domain queried for its TXT record.'}),
                     ('txt', ('str', {}), {'ro': 1,
                          'doc': 'The string returned in the TXT record.'}),
                 )),
-                ('inet:dns:req', {}, (
-                    ('client', ('inet:client', {}), {
-                        'ro': True,
-                        'doc': 'The inet:addr which requested the FQDN',
-                    }),
-                    ('client:ipv4', ('inet:ipv4', {}), {
-                        'ro': True,
-                        'doc': 'The IPv4 of the client.',
-                    }),
-                    ('client:ipv6', ('inet:ipv6', {}), {
-                        'ro': True,
-                        'doc': 'The IPv6 of the client.',
-                    }),
-                    ('fqdn', ('inet:fqdn', {}), {
-                        'ro': True,
-                        'doc': 'The requested FQDN',
-                    }),
-                    ('type', ('inet:dns:type', {}), {
-                        'ro': True,
-                        'doc': 'The type of DNS record requested.',
-                    }),
 
+                ('inet:dns:query', {}, (
```

Comment:
Is this meant as the "fused" (rolled up) version of individual inet:dns:requests? If so should it have the equivalent of optional seen:min / seeen:max (first known time client made the query / most recent time client made the query)?

Reply:
:+1: